packages/restlessness-core/src/JsonEndpoints/index.ts

```diff
@@ -124,7 +124,18 @@ class JsonEndpoints extends JsonConfigFile<JsonEndpointsEntry> {
     // Re-generate exporter file with considering the new endpoint
     await this.generateExporter();
 
-    // Add e new function handler in serverless.json read by serverless.yml
+    // Add e new function handler in serverless offline json that will be used for local development
+    // It also adds the authorizer handler, if it doesn't exist yet
+    await JsonServices.addEndpoint({
```

Comment:
Could it be better to handle the offline service inside JsonServices? For example addEndpoint automatically create the endpoint for the specified service and also for the offline one.

I think this may be necessary since some operation requires some logic. For example if we add an endpoint with a shared authorizer

Reply:
Actually It is inside JsonServices, but I think I got your point. I was thinking about treating offline as an other service, but it's true that some logics change, so maybe it's better to use the JsonServices class as a wrapper, agree